app/views/vendors/products.html.erb

```diff
@@ -0,0 +1,29 @@
+<% @products = Product.where(vendor_id: @vendor.id) %>
```

Comment:
Ideally you should define any instance variables like this in your controller method instead of in the view. It occurred to me that you may not have covered this yet, but this is another place where you can use @vendor.products to make things a little easier. Active Record associations are the bomb! You can use them for more specific queries too, like @vendor.products.first or maybe if your vendor had an attribute for multiple locations, you could do something like @vendor.products.where(location: "Seattle")